docs/api/static-transfers.md

```diff
@@ -0,0 +1,79 @@
+# Static Transfer API
+
+The Static transfer API is a non-interactive API for downloading LFS objects.
+Its main intent is to provide simple interoperability with existing data
+storage solutions by offloading serving LFS objects to them while using
+git and git LFS for version management.
+
+Support for it by Git LFS clients and servers is optional.
+
+Git LFS clients MAY implement this transfer API in a fallback manner:
+first attempting to download the LFS object using the Static transfer
+API and if this fails (e.g., the manifest file does not exist,
+oid cannot be found in the manifest file) use the Basic transfer API.
+
+## Downloads
+
+Downloading an object first requires fetching a manifest file from using
+HTTP from the remote server. The proposed location of the manifest file
+is `https://<remote domain>/.well-known/git-lfs-manifest.json`. The
+location is configured using the `lfs.staticurl` configuration option,
+or remote-specific `remote.{name}.lfsstaticurl` configuration option.
+Both of them can be provided in `.lfsconfig` file as well.
+
+The manifest file MUST contain a JSON payload that looks like this:
+
+```json
+{
+  "version": "1",
+  "transfer": "static",
+  "objects": [
+    {
+      "oid": "1111111",
+      "size": 123,
+      "authenticated": true,
+      "actions": {
+        "download": {
+          "href": "https://some-download.com/1111111",
+          "header": {
+            "Authorization": "Basic ..."
+          },
+          "expires_at": "2016-11-10T15:29:07Z"
+        }
+      }
+    }
+  ]
+}
+```
+
+The remote server SHOULD return together with the contents of the
+manifest file also suitable caching response headers (e.g.,
+`Cache-Control`, `Pragma`, `Last-Modified`, `ETag`). Moreover, it SHOULD
+respond to GET and HEAD HTTP requests and support caching request
+headers (e.g., `If-Modified-Since`, `If-None-Match`). Git LFS client
+SHOULD attempt to fetch the manifest file again if any of the
+authorization tokens expires.
```

Comment:
My strong preference is that we don't have expiring tokens.  If we need expiring tokens, then we need a server to handle that properly, especially since caching can prevent re-fetching from working.

Let's also recommend that it support proper Range headers.

Reply:
Sounds good. I can remove expiration from the spec and mention range headers.